frontend/src/component/admin/network/NetworkOverview/NetworkOverview.tsx

```diff
@@ -22,24 +22,20 @@ export const NetworkOverview = () => {
     const { metrics } = useInstanceMetrics();
 
     const apps: INetworkApp[] = [];
-    if (Boolean(metrics)) {
-        Object.keys(metrics).forEach(metric => {
-            apps.push(
-                ...(
-                    metrics[metric].data?.result
-                        ?.map(result => ({
-                            label: result.metric?.appName,
-                            reqs: parseFloat(
-                                result.values?.[
-                                    result.values?.length - 1
-                                ][1].toString() || '0'
-                            ).toFixed(2),
-                            type: metric.split('Metrics')[0],
-                        }))
-                        .filter(app => app.label !== 'undefined') || []
```

Comment:
Just a personal opinion, but I believe we should also show `undefined` ones

Reply:
![Screenshot from 2022-12-19 11-31-15](https://user-images.githubusercontent.com/455064/208405519-28ce6d8d-6038-4990-a63b-750ee93f19e9.png)
![Screenshot from 2022-12-19 11-31-00](https://user-images.githubusercontent.com/455064/208405521-20c6a8ca-feb5-4577-9d30-70c6a3d4040a.png)

Reply:
For now I'm not showing 'undefined' (should we renamed to 'unknown' if we want to show it), just to avoid changing  the current behavior